gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts

```diff
@@ -61,47 +63,54 @@ export default class OCR2InitializeFlow extends FlowCommand<TransactionResponse>
         },
       },
       {
-        name: 'Begin Offchain Config',
-        command: BeginOffchainConfig,
+        name: 'Set Billing',
+        command: SetBilling,
         flags: {
           state: FlowCommand.ID.contract(this.stepIds.OCR_2),
-          version: 2,
         },
       },
       {
-        name: 'Write Offchain Config',
-        command: WriteOffchainConfig,
+        id: this.stepIds.PROPOSAL,
+        name: 'Create Proposal',
+        command: CreateProposal,
+      },
+      {
+        name: 'Propose Config',
+        command: ProposeConfig,
         flags: {
           state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+          proposalId: FlowCommand.ID.data(this.stepIds.PROPOSAL, 'proposal'),
         },
       },
       {
-        name: 'Commit Offchain Config',
-        command: CommitOffchainConfig,
+        name: 'Propose Offchain Config',
+        command: ProposeOffchainConfig,
         flags: {
           state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+          proposalId: FlowCommand.ID.data(this.stepIds.PROPOSAL, 'proposal'),
         },
       },
       {
-        name: 'Set Config',
-        command: SetConfig,
+        name: 'Propose Payees',
+        command: ProposePayees,
         flags: {
           state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+          proposalId: FlowCommand.ID.data(this.stepIds.PROPOSAL, 'proposal'),
         },
       },
       {
-        name: 'Set Payees',
-        command: SetPayees,
+        name: 'Finalize Proposal',
+        command: FinalizeProposal,
         flags: {
-          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
-          link: process.env.LINK || this.flags.link,
+          proposalId: FlowCommand.ID.data(this.stepIds.PROPOSAL, 'proposal'),
         },
       },
       {
-        name: 'Set Billing',
-        command: SetBilling,
+        name: 'Accept Proposal',
```

Comment:
Note: For later you'll want to be able to create and finalize a proposal, then do the accept step via multisig

Reply:
Right. This would be useful for the initial deployment only, before the transfer